test/framework/src/main/java/org/elasticsearch/repositories/blobstore/BlobStoreTestUtil.java

```diff
@@ -129,7 +129,7 @@ public static void assertConsistency(BlobStoreRepository repository, Executor ex
                 try (InputStream blob = blobContainer.readBlob("index-" + latestGen);
                      XContentParser parser = XContentType.JSON.xContent().createParser(NamedXContentRegistry.EMPTY,
                          LoggingDeprecationHandler.INSTANCE, blob)) {
-                    repositoryData = RepositoryData.snapshotsFromXContent(parser, latestGen);
+                    repositoryData = RepositoryData.snapshotsFromXContent(parser, latestGen, true);
```

Comment:
why `true` here?

Reply:
Right my bad no reason to be lenient here ... moved to `false`.